jax/_src/numpy/lax_numpy.py

```diff
@@ -1958,6 +1958,14 @@ def block(arrays: ArrayLike | list[ArrayLike]) -> Array:
   out, _ = _block(arrays)
   return out
 
+@overload
+def atleast_1d(x: ArrayLike, /) -> Array:
+  ...
+
+@overload
+def atleast_1d(x: ArrayLike, y: ArrayLike, /, *arys: ArrayLike) -> list[Array]:
+  ...
+
```

Comment:
Split out these changes in a separate PR?

Reply:
Yes, okay, I will do that when we get close to having this checked in.   This PR will expose a lot of typing errors as code that's jitted becomes typed.

Reply:
Factored out as requested: https://github.com/google/jax/pull/18395